filter/filter_impl/graceful_shutdown_filter.go

```diff
@@ -61,7 +55,7 @@ func (gf *gracefulShutdownFilter) Invoke(ctx context.Context, invoker protocol.I
 	}
 	atomic.AddInt32(&gf.activeCount, 1)
 	if gf.shutdownConfig != nil && gf.activeCount > 0 {
-		gf.shutdownConfig.RequestsFinished = false
+		gf.shutdownConfig.RequestsFinished.Store(false)
```

Comment:
Atomic use LOCK CPU instruction, may cause performance issues.

Reply:
IMO, CAS lock is the best choice in high concurrency scenario, because using `sync.Mutex` may block the Goroutine and results in greater performance losses.